terraform/application/application.tf

```diff
@@ -0,0 +1,40 @@
+module "application_configuration" {
+  source = "./vendor/modules/aks//aks/application_configuration"
+
+  namespace              = var.namespace
+  environment            = var.environment
+  azure_resource_prefix  = var.azure_resource_prefix
+  service_short          = var.service_short
+  config_short           = var.config_short
+  secret_key_vault_short = "app"
+
+  is_rails_application = true
+
+  config_variables = merge(
+    local.app_env_values,
+    {
+      ENVIRONMENT_NAME = var.environment
```

Comment:
Formatting. I suggest [adding fmt and tflint](https://github.com/DFE-Digital/teacher-services-cloud/blob/main/.github/actions/validate-terraform/action.yml#L35) (but it can be in a different PR)